testeng/jobs/cleanupPythonCode.groovy

```diff
@@ -0,0 +1,85 @@
+
+def PYTHON_VERSION = 'python3.5'
+
+def DEFAULT_BRANCH = 'master'
+
+def ORG = 'edx'
+
+def githubUserReviewers = ['']
+
+def githubTeamReviewers = ['arbi-bom']
+
+def emails = ['arbi-bom@edx.org']
+
+
+job('cleanup-python-code-job'){
+
+ parameters {
+        stringParam('repoName', null, 'URL of the target repository')
```

Comment:
Shouldn't the help text say it's the name of the target repository?

Reply:
yes, it was based on my early understanding (to expect full repo URL from the user)